week-2/errors/1.js

```diff
@@ -1,13 +1,20 @@
 // Predict and explain first...
+
+//ans ; we declared const decimalNumber inside the body function it mean it is Local scope, which never allow variable to call out side the function.
+// as a result we face syntexError.
 // Write down the error you predict will be raised
+//ans; syntaxError
 // Why will an error occur when this program runs?
+//ans: Variable Scope: If the variable is defined within the function using the var, let, or const keyword, it is only accessible within that function's scope.
```

Comment:
This was less to do with accessing a variable outside of its scope, and more to do with creating the variable twice. We first declared the variable as an argument (inside the function declaration), and then again as a `const` variable inside the function body. We can't use the same variable name twice in the same scope, therefore we get the error